analytics/views.py

```diff
@@ -50,6 +51,7 @@ def render_stats(request: HttpRequest, realm: Realm) -> HttpRequest:
                                page_params=JSONEncoderForHTML().encode(page_params)))
 
 @zulip_login_required
+@zulip_otp_required(login_url=settings.HOME_NOT_LOGGED_IN)
```

Comment:
Do we need to add this everywhere we use `zulip_login_required` in the future?  If so, would it make sense to just have `zulip_login_required` handle the OTP case as well?

Reply:
Yep, done.